File: ChatApp.WebApp/src/application/models/response/UserSessionResponse.ts
export type UserTokenResponse = {
	AccessToken: string;
	RefreshToken: string;
};

export type UserSessionResponse = {
	UserId: string;
	UserStatus: number;
	FirstName: string;
	LastName: string;
	EmailAddress: string;
	Username: string;
	LanguageCode: string;
	UserDateCreated: Date;
	IsAdmin: boolean;
	AccessToken: string;
	RefreshToken: string;
};

File: ChatApp.Application/Models/Responses/UserSessionResponse.cs
namespace ChatApp.Application.Models.Responses
{
    public class UserSessionResponse
    {
        public Guid? UserId { get; set; }
        public int? UserStatus { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? EmailAddress { get; set; }
        public string? Username { get; set; }
        public string? LanguageCode { get; set; }
        public DateTime? UserDateCreated { get; set; }
        public bool? IsAdmin { get; set; }
        public string? AccessToken { get; set; }
        public string? RefreshToken { get; set; }
    }
}
